README.org

```diff
@@ -77,9 +88,11 @@ Advertise your module here, open a PR and include a org-mode link!
 - [[./media/amixer/README.org][amixer]] :: Manipulate the volume using amixer
 - [[./media/stump-radio/README.org][stump-radio]] :: Minimalistic mplayer-based radio for StumpWM.
 - [[./media/stump-volume-control/README.org][stump-volume-control]] :: Minimalistic amixer-based volume control for StumpWM.
+
```

Comment:
These newlines are inserted by [./update-readme.sh.
](https://github.com/stumpwm/stumpwm-contrib/blob/6514148c7ced697cf949eaaabf8312b9f061f595/update-readme.sh#L10-L13) We need to update the shell script. Otherwise they will be gone the next time `update-readme.sh` is run.. Something like the following should work.

```
awk '/\/util\// && !x {print "** Utilities"; x=1} 1' tmpReadme > tmp; mv tmp tmpReadme
awk '/\/media\// && !x {print "\n** Media"; x=1} 1' tmpReadme > tmp; mv tmp tmpReadme
awk '/\/minor-mode\// && !x {print "\n** Minor Modes"; x=1} 1' tmpReadme > tmp; mv tmp tmpReadme
awk '/\/modeline\// && !x {print "\n** Modeline"; x=1} 1' tmpReadme > tmp; mv tmp tmpReadme
```

Reply:
You are, of course, correct. Apologies.